Gemfile

```diff
@@ -6,4 +6,10 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 # Declare your gem's dependencies in topological_inventory.gemspec.
 # Bundler will treat runtime dependencies like base dependencies, and
 # development dependencies will be added by default to the :development group.
+gem 'rbac-api-client', :git => "https://github.com/RedHatInsights/insights-rbac-api-client-ruby", :branch => "master"
+
+group :development, :test do
+  gem 'webmock'
```

Comment:
Should this be in the gemspec as a development dependency?

Reply:
If it's in the gemspec you should then remove this line?

Reply:
Removed.